Source/Private/ASTextNode2.mm

```diff
@@ -488,7 +488,54 @@ - (id)_linkAttributeValueAtPoint:(CGPoint)point
    inAdditionalTruncationMessage:(out BOOL *)inAdditionalTruncationMessageOut
                  forHighlighting:(BOOL)highlighting
 {
-  AS_TEXT_ALERT_UNIMPLEMENTED_FEATURE();
+  ASDN::MutexLocker l(__instanceLock__);
+
+  ASTextContainer *containerCopy = [_textContainer copy];
+  containerCopy.size = self.calculatedSize;
+  ASTextLayout *layout = [ASTextNode2 compatibleLayoutWithContainer:containerCopy text:_attributedText];
+  NSRange visibleRange = layout.visibleRange;
+  NSRange clampedRange = NSIntersectionRange(visibleRange, NSMakeRange(0, _attributedText.length));
+
+  ASTextRange *range = [layout closestTextRangeAtPoint:point];
+
+  // For now, assume that a tap inside this text, but outside the text range is a tap on the
+  // truncation token.
+  if (![layout textRangeAtPoint:point]) {
+    *inAdditionalTruncationMessageOut = YES;
+    return nil;
+  }
+
+  NSRange effectiveRange = NSMakeRange(0, 0);
+  for (NSString *attributeName in self.linkAttributeNames) {
+    id value = [self.attributedText attribute:attributeName atIndex:range.start.offset longestEffectiveRange:&effectiveRange inRange:clampedRange];
+    NSString *name = attributeName;
```

Comment:
Done. Doing this required adding a semi-ugly __strong to the iteration variable declaration, as the compiler does generate an error for writing to that variable without it.